core/unix/include/sigcontext.h

```diff
@@ -43,7 +31,7 @@
  * extended state information in the memory layout pointed by the fpstate
  * pointer in sigcontext.
  */
-struct _fpx_sw_bytes {
+struct kernel_fpx_sw_bytes {
```

Comment:
So: typedef struct _kernel_fpx_sw_bytes_t { ... } kernel_fpx_sw_bytes_t;
and use "kernel_fpx_sw_bytes_t" in other places? I don't see why not...